<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Using String Notation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Using Functor Regular Notation</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Using Functor Lambda  Notation</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
